### DISCUSSION TITLE
DNS error

### DISCUSSION THREAD

**Discussion by WorkMender** [vercel/community#6006] [Help] [answered]
### Summary

I have connected my custom domain and the status seems to be ok however I am unable to access the app via my custom domain. This just started about 2 hours ago. Whereas it was working fine about 12 hours ago. The website error is DNS_PROBE_FINISHED_NXDOMAIN

### Example

workmender.com

### Steps to Reproduce

There are no steps to reproduce the error. However I am using Namecheap. The access to the website stopped unexpectedly for my root domain and subdomains

**Comment by amyegan** [comment]
Thanks for reaching out @WorkMender. `NXDOMAIN` means non-existent domain. So the error means the DNS query couldn't locate the domain. It may be caused by VPN or security software altering network settings, IPS issues, domain expiration, or misconfigured DNS settings. 

We have a doc about [troubleshooting domains](https://vercel.com/docs/projects/domains/troubleshooting) that offers some options for digging into domain issues. If you're able to share the domain here, someone may be able to help you dig into the details and uncover the problem.

**Reply by WorkMender** [reply]
The domain is workmender.com. The domain is being hosted with namecheap. Thank you.

**Reply by amyegan** [reply]
Thanks for sharing that. I was able to access the site without running into a `DNS_PROBE_FINISHED_NXDOMAIN` error. And the [A](https://www.whatsmydns.net/#A/workmender.com) and [CNAME](https://www.whatsmydns.net/#CNAME/www.workmender.com) records look correct.

I suspect the error is being caused by a network issue from your ISP, VPN, anti-virus, or other security software. Are you able to access the site from a different network?

**Reply by WorkMender** [reply]
No I am unable to

**Reply by iamkalio** [reply]
@amyegan It seems @WorkMender and I are facing similar Issue, my website is https://www.rowlandadanmascholarship.ng/, I haven't altered any panel configuration and it seems I cant load it up anymore, it did work a couple hours ago. i have reached out to the domain providers and it seems the issue is not from their end. please how do we resolve this?

**Reply by WorkMender** [reply, marked as answer]
I was able to fix the issue. My issue had to do with duplicate hosts on CNAME and A Record. Possibly check if you have any duplicates in these records.

**Comment by iamkalio** [comment]
thanks for the feedback I'll check it right away.
